include/SFML/Audio/InputSoundFile.hpp

```diff
@@ -63,8 +63,9 @@ class SFML_AUDIO_API InputSoundFile : NonCopyable
     ////////////////////////////////////////////////////////////
     /// \brief Open a sound file from the disk for reading
     ///
-    /// The supported audio formats are: WAV (PCM only), OGG/Vorbis, FLAC.
+    /// The supported audio formats are: WAV (PCM only), OGG/Vorbis, FLAC, MP3.
     /// The supported sample sizes for FLAC and WAV are 8, 16, 24 and 32 bit.
+    /// For MP3 files with big (>16kb) APEv2 tag there low chance of garbage decoded at the end.
```

Comment:
The verb is missing: `... there is a low change ...`

Reply:
Thanks) Updated the PR.